salt/modules/zypper.py

```diff
@@ -1195,6 +1206,7 @@ def install(name=None,
         __zypper__(no_repo_failure=ignore_repo_failure).call(*cmd)
 
     __context__.pop('pkg.list_pkgs', None)
+    __context__.pop('pkg.list_list_provides', None)
```

Comment:
Ah, these are invalidators. OK, can we move that into one variable (uppercase, class-level outside the `__init__`) and call them like `CACHED_LIST_PROVIDES` or something?

Reply:
I just copied the same what we do for 'pkg.list_pkgs'. Why is this now not ok?
I am a fan of doing things like others do and not invent own coding style.
Grepping over the code show that everybody is doing it this way. So why should we now do something which is different to what all the others are doing?

I would say either change this in the whole code or not at all. But this is nothing for this PR.

Reply:
That was not OK either :wink: I would actually move that away to the variable as well.